cdap-ui/app/cdap/components/Administration/AdminConfigTabContent/SystemProfilesAccordion/index.js

```diff
@@ -91,10 +77,19 @@ export default class SystemProfilesAccordion extends Component {
         >
           {T.translate(`${PREFIX}.create`)}
         </Link>
-        <ProfilesListView
-          namespace='system'
-          onChange={this.onChange}
-        />
+        <Label
+          className="import-profile-label"
+          for="import-profile"
+        >
+          {T.translate(`${PREFIX}.import`)}
+          <Input
+            type="file"
+            accept='.json'
+            id="import-profile"
+            onChange={importProfile.bind(this, 'system')}
```

Comment:
no value to display here?

Reply:
Do you mean displaying something like the name of the file that was uploaded? So if the file is valid, then we add it to the list of profiles right away, if not valid then we show an error, so I don't think we need to show the file name here